src/Gui/DlgMaterialPropertiesImp.cpp

```diff
@@ -81,10 +81,10 @@ void DlgMaterialPropertiesImp::on_ambientColor_changed()
     float b = (float)col.blue()/255.0f;
     App::Color ambient(r,g,b);
 
-    for (std::vector<ViewProvider*>::iterator it= Objects.begin();it!=Objects.end();++it) {
+    for (std::vector<ViewProvider*>::iterator it= Objects.begin(); it != Objects.end(); ++it) {

```

Comment:
Why isn't this one changed?

Reply:
Because the type is hidden in instance variable `Objects` whose definition is not immediately obvious.